<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="LE_filepath"/>
        </item>
        <item>
         <widget class="QPushButton" name="PBTN_OpenFile">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBTN_Compile">
          <property name="text">
           <string>编译</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBTN_Run">
          <property name="text">
           <string>运行</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="PBTN_Token">
          <property name="text">
           <string>词法分析</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBTN_SyntaxTree">
          <property name="text">
           <string>语法树</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBTN_Symbol">
          <property name="text">
           <string>符号表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PBTN_IR">
          <property name="text">
           <string>中间代码</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="VLayout_SourceCode">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="LB_SourceCode">
            <property name="text">
             <string>源文件：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="TB_SourceCode"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="VLayout_output">
          <item>
           <widget class="QLabel" name="LB_Output">
            <property name="text">
             <string>输出：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="TB_Output"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
